### PR TITLE
Spinus issue 139 gherkin terminal reporter

### PR DESCRIPTION
**Pull request by olegpidsadnyi**
Still couple of tests are failing

### CODE REVIEW

**Comment by olegpidsadnyi** [comment]
@spinus please pull and have a look. There are still couple of tests failing

**Comment by olegpidsadnyi** [comment]
It passes with tox, however fails comparing regular and gherkin execution with "1 passed" vs "2 passed".

**Comment by spinus** [comment]
Hey, thanks for this. I'll be fixing it tomorrow.

**Comment by spinus** [comment]
@olegpidsadnyi I was looking at your comment, but not sure what do you mean.

**Comment by olegpidsadnyi** [comment]
I don't know.. If I run this file alone as
py.test tests/feature/test_gherkin_terminal_reporter.py

it fails. passes CI though..

    @then("output must be formatted the same way as regular one")
    def output_must_be_the_same_as_regular_reporter(test_execution):
        reg = test_execution['regular']
        ghe = test_execution['gherkin']
        assert reg.ret == 0
        assert ghe.ret == 0
        #  last line can be different because of test execution time is printed
        reg_lines = reg.stdout.lines if reg.stdout.lines[-1] else reg.stdout.lines[:-2]
        reg_lines[-1] = re.sub(r' \d+\.\d+ ', ' X ', reg_lines[-1])
        ghe_lines = ghe.stdout.lines if ghe.stdout.lines[-1] else ghe.stdout.lines[:-2]
        ghe_lines[-1] = re.sub(r' \d+\.\d+ ', ' X ', ghe_lines[-1])
        for l1, l2 in zip(reg_lines, ghe_lines):
>           assert l1 == l2
E           assert '============...=============' == '=============...============='
E             - =========================== 1 passed in X seconds ===========================
E             ?                             ^
E             + =========================== 2 passed in X seconds ===========================
E             ?                             ^

**Comment by spinus** [comment]
Oh ok, I see. I couldn't reproduce it locally as well that's why I was a little confused.
Can you paste your packages versions (I'll try to reproduce it)? Looks like one test with gherkin reported is called twice.

**Comment by olegpidsadnyi** [comment]
I've put that line (self.stats.setdefault(cat, []).append(rep)) all the way down. Not sure if it is correct now, at least it doesn't duplicate the stats anymore and test is passing.

**Comment by spinus** [comment]
Thank you for fixing that.

**Comment by spinus** [comment]
As you wrote to me, I updated readme and changes file.

File: pytest_bdd/gherkin_terminal_reporter.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+        )
+    )
+
+
+def configure(config):
+    if config.option.gherkin_terminal_reporter:
+        # Get the standard terminal reporter plugin and replace it with our
+        current_reporter = config.pluginmanager.getplugin('terminalreporter')
+        if current_reporter.__class__ != TerminalReporter:
+            raise Exception("gherkin-terminal-reporter is not compatibile with any other terminal reporter."
```
Comment (The-Compiler) [inline pytest_bdd/gherkin_terminal_reporter.py]: nitpick: compatibile -> compatible

**Comment by olegpidsadnyi** [comment]
Fixed in #139

**Pull request closed without merging**